src/components/Certificate/certificate.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { beforeEach, describe, expect, test, vi } from "vitest";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { MemoryRouter, Route, Routes } from "react-router-dom";
+import Certificate from "./certificate";
+
+const username = "testUser";
+const pathname = `/members/${username}/certificate`;
+
+describe("Certificate Page", () => {
+  beforeEach(() =>
+    render(
+      <MemoryRouter initialEntries={[pathname]}>
+        <Routes>
+          <Route
+            path="/members/:username/certificate"
+            element={<Certificate />}
+          />
+        </Routes>
+      </MemoryRouter>,
+    ),
+  );
+
+  test("render the correct title", () => {
```

Comment:
여러 테스트 케이스를 `describe()`로 묶을 때는 보통 `it()`을 사용하지 않나요? 묶지 않을 때는 `test()`를 사용하고요. 
그리고 당연한 얘기니 `correct`은 빼도 될 것 같습니다.

```suggestion
  it("renders the title", () => {
```

Reply:
오.. 그렇군요! 제가 느끼기에는 관용적 맥락인거 같네요. 그리고 문서에서나 타입정의된걸 확인하니 동작은 동일하지만 네이밍이 다르네요.
타입정의된 순서도 test, describe, it순서로 정의되어있는데 이는 개발자가 보통 describe, it을 커플링으로 사용하기에 묶어논거 같네요.
달레님 말씀대로 it으로 변경하면 좋을거 같습니다!